eng/Versions.props

```diff
@@ -266,5 +266,7 @@
       rather explicitly override it.
     -->
     <UsingToolMicrosoftNetCompilers Condition="'$(BootstrapBuildPath)' == ''">true</UsingToolMicrosoftNetCompilers>
+
+    <UseVSTestRunner>true</UseVSTestRunner>
```

Comment:
do we also need to update `MicrosoftNETTestSdkVersion` in this file?

Reply:
What relationship does that have to how the tests run here?

Reply:
I had the impression that `Microsoft.Net.Test.Sdk` actually controls what version of the test host gets loaded and that updating it is required to e.g. benefit from the change that makes the test host actually multi-target (or whatever was blocking us from moving to `dotnet test` before).